VSCode/armsnippets.json

```diff
@@ -1489,4 +1489,52 @@
         "   }"
     ],
     "description": "Azure Kubernetes Service Cluster"
+},
+"Azure Logic App": {
+    "prefix": "arm-logicapp",
+    "body": [
+      "{",
```

Comment:
Indentation is still off a bit - lines in the snippet body should be indented 4 spaces. Also lines added to this file aren't indented the same as the rest of the file. VSCode can help autoformat the file for you.